src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1109,6 +1109,14 @@ private void WorkspaceAdded(WorkspaceModel item)
             else
             {
                 var newVm = new WorkspaceViewModel(item, this);
+
+                // For Json Workspaces, workspace view info need to be read agin from file
+                string fileContents;
+                if (DynamoUtilities.PathHelper.isValidJson(newVm.Model.FileName, out fileContents))
+                {
+                    ExtraWorkspaceViewInfo viewInfo = WorkspaceViewModel.ExtraWorkspaceViewInfoFromJson(fileContents);
```

Comment:
this could potentially be a lot of data, wonder if it's worth cacheing, lets just watch this for performance issues.

Reply:
Right I guess I only care about node positions in this case? Maybe other view properties are important as well like scalefactor, etc. So I am sticking to deserialize them all for now